File: Java/estruturaDeDecisao/If.java
/*
As instruções if nos permitem executar código com condições.

Por exemplo, vamos analisar o seguinte código:

int age = 20;
String status = "Child";
if (age > 18) {
    status = "Adult";
}
age += 1;
O código acima verifica se a variável age é maior que 18. Se for, ele definirá status para conter a string "Adult".

No final, o código incrementará age por 1, independentemente de a idade ser maior que 18 ou não.



Para usar uma instrução if em Java, precisamos usar chaves {} para definir o bloco de código, e tudo dentro da instrução if deve ser colocado entre essas chaves:

if (condition) {
    code;
    code;
    code;
}
Se a condição for true, entraremos no bloco de código dentro do if (O código indentado)


Desafio

Iniciante
Você recebe um código.

As variáveis a e b possuem valores ausentes, preencha-as para que o código dentro da declaração if seja executado e c seja igual a 3.

Bônus: tente encontrar mais de uma solução!
 */
public class Main {
    public static void main(String[] args) {
        int a = ?
        int b = ?

        // Don't change below this line
        int c = 0;
        if (a >= b && !(b < 10)) {
            c = 2;
        }

        c += 1;
        System.out.println("c = " + c);
    }
}

File: JavaScript/Coddy/TomadaDeDecisao/IF.js
/*
Instruções if nos permitem executar código com condições.

Por exemplo, vejamos o seguinte código:

let age = 20;
let status = "Child";
if (age > 18) {
	status = "Adult";
}
age += 1;
O código acima verifica se a variável age é maior que 18. Se for, ele definirá status para conter a string "Adult".

No fim, o código incrementará age em 1, independentemente de a idade ser maior que 18 ou não.

 

Para usar uma instrução if, precisamos adicionar parênteses () que determinarão a condição, e tudo o que estiver dentro do if fica entre chaves {}:

if (condition) {
    code;
    code;
    code;
}
Se a condição for verdadeira, entraremos no bloco de código dentro do if (o código dentro das chaves)

challenge icon
Desafio

Iniciante
Você recebeu um código.

As variáveis a e b têm valores ausentes, preencha-os para que o código dentro da instrução if seja executado!

Bônus: tente encontrar mais de uma solução!
*/
let a = 100
let b = 50

// Don't change below this line
let c = 0;
if (a >= b && !(b < 10)) {
    c = 2;
}

c += 1;
console.log(`c = ${c}`);